doc/data-mining-library/source/tutorial/data.rst

```diff
@@ -60,6 +60,59 @@ The following script wraps-up everything we have done so far and lists first 5 d
 
 Note that data is an object that holds both the data and information on the domain. We show above how to access attribute and class names, but there is much more information there, including that on feature type, set of values for categorical features, and other.
 
+Creating a Data Table
+---------------------
+
+To create a data table from scratch, one needs two things, a domain and the data. The domain is the description of the variables, i.e. column names, types, roles, etc.
+
+First, we create the said domain. We will create three types of variable, numeric (ContiniousVariable), categorical (DiscreteVariable) and text (StringVariable). Numeric and categorical variables will be used a features (also known as X), while the text variable will be used as a meta variable.
+
+    >>> from Orange.data import Domain, ContinuousVariable, DiscreteVariable, StringVariable
+    >>> 
+    >>> domain = Domain([ContinuousVariable("var1"), DiscreteVariable("var2", values=["red", "blue"])], metas=[StringVariable("var3")])
+
+Now, we will build the data with numpy.
+
+    >>> import numpy as np
+    >>> 
+    >>> var1 = np.array([1.2, 1.4, 1.5, 1.1, 1.2])
+    >>> var2 = np.array([0, 1, 1, 1, 0])
+    >>> var3 = np.array(["U13", "U14", "U15", "U16", "U17"], dtype=object)
+
+Two things to note here. var2 has values 0 and 1, even though we specified it will be a categorical variable with values "red" and "blue". X (features in the data) can only be numbers, so the numpy matrix will contain numbers, while Orange will handle the categorical representation internally. 0 will be mapped to the value "red" and 1 to "blue" (in the order, specified in the domain).
+
+Text variable requires dtype=object for numpy to handle it correctly.
+
+Next, variables have to be transformed to columns (lists of lists). 
+
+    >>> var1 = var1.reshape(-1, 1)
+    >>> var2 = var2.reshape(-1, 1)
+    >>> var3 = var3.reshape(-1, 1)
+
+Finally, we create a table. We need a domain and variables, which can be passed as X (features), Y (class variable) or metas.
+
+    >>> table = Table.from_numpy(domain, X=X, metas=var3)
```

Comment:
Where did X come from? `X = np.hstack((var1, var2, var3))`

However, there's no need to reshape. `X = np.column_stack((var1, var2, var3))` will stack 1-d arrays into columns.

Reply:
Thanks, I was wondering which one is more appropriate to us. I have always been using hstack, but if column_stack does the trick, even better!